File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0){
            return false;
        }
        int count =0;
        int temp = x;
        while(temp != 0){
            int a = temp%10;
            count = count*10+a;
            temp /= 10;
        }
        return (count == x);
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        if x<0:
            return False
        count =0
        temp = x
        while temp != 0:
            a = temp%10
            count = count*10+a
            temp /= 10
        return count == x

        """
        :type x: int
        :rtype: bool
        """
        